java/code/src/com/suse/manager/webui/websocket/VirtNotifications.java

```diff
@@ -127,41 +128,51 @@ public void onMessage(Session session, String messageBody) {
         // Each session sends messages to tell us what action ID they need to monitor
         Set<Long> serverIds = wsSessions.get(session);
         if (serverIds != null) {
-            Long newId = null;
-            try {
-                newId = Long.parseLong(messageBody);
-                serverIds.add(newId);
+            Optional<User> userOpt = Optional.ofNullable(session.getUserProperties().get("webUserID"))
+                    .map(webUserID -> {
+                        User innerUser = UserFactory.lookupById((Long) webUserID);
+                        innerUser.getRoles();
```

Comment:
Is this line needed? Why?

Reply:
It's not needed. It was a copy/past that was left to clean. :+1: 
I will make a commit with this code cleanup